examples/python/evm/get_or_create_account.py

```diff
@@ -10,23 +10,28 @@
 
 async def main():
     async with CdpClient() as cdp:
-        # Single account creation example
-        name = "Account1"
-        account = await cdp.evm.get_or_create_account(name=name)
-        print("Account Address: ", account.address)
-        account2 = await cdp.evm.get_or_create_account(name=name)
-        print("Account 2 Address: ", account2.address)
-        print("Are accounts equal? ", account.address == account2.address)
-
-        # Concurrent account creation example
-        account_coros = [
-            cdp.evm.get_or_create_account(name="Account"),
-            cdp.evm.get_or_create_account(name="Account"),
-            cdp.evm.get_or_create_account(name="Account"),
-        ]
-        accounts = await asyncio.gather(*account_coros)
-        for i, acc in enumerate(accounts, 1):
-            print(f"EVM Account Address {i + 1}: ", acc.address)
+        # First create an owner account
+        owner = await cdp.evm.create_account()
+        print("Created owner account:", owner.address)
+
+        # Get or create a smart account with the owner
+        # Note: Each owner can only have one smart account
+        account = await cdp.evm.get_or_create_smart_account(name="MyAccount", owner=owner)
```

Comment:
this example file is about server accounts, not smart accounts. it should not change at all